python/test_file_generator.py

```diff
@@ -203,8 +209,9 @@ def v607_file(self):
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v608_file(self):
```

Comment:
This edit produces an upload error - rather than a v608 fail. May need to discuss.

Reply:
Is the parser checking length of the ULI?